web/pages/_document.tsx

```diff
@@ -32,6 +32,10 @@ export default function Document() {
           `,
           }}
         />
+        <script
+          src="https://cdn.jsdelivr.net/npm/link-summoner@1.0.2/dist/browser.min.js"
+          type="module"
+        ></script>
```

Comment:
I can tell you that `yarn lint` is gonna complain you aren't using Next's `Script` component: 

https://nextjs.org/docs/basic-features/script

But I agree with Stvad that I don't know why this shouldn't be bundled.

Reply:
Dumb question, how do we bundle it? @Stvad maybe worth adding instructions to the README?

I was going to try `yarn add link-summoner` and then... try `import "link-summoner"` from the tsx files where we want popups to appear??

Reply:
That sounds about right (may need to add specific subpath, but probably not). will add instructions Tomorrow/Thu

Reply:
Hrm yeah I tried the naive thing I described with `import 'link-summoner'` in _app.tsx, and I just got this: 

![image](https://user-images.githubusercontent.com/856034/167846245-4daf1287-b017-4948-a792-191b73423ae2.png)

Not sure if this is a NextJS specific issue (where `document` is not available on server-side rendered pages, and thus causes an error), or I'm doing something wrong with the import.

Reply:
Tested things on my side (pushed a bundling example in https://github.com/Stvad/link-summoner/tree/master/examples/bundling) 

Pretty sure it's Next.js SSR stuff. 

I was planning on having a version that you would use a function (allowing to supply additional parameters, etc). I think that would allow you to invoke it only if `document` exists.

Reply:
with version 1.1.0 this should work as follows:

```
import {initPreviews} from "link-summoner"

// can be wrapped into if(typeof document !== undefined)
initPreviews()
```

yet to properly update docs

Reply:
Updated the docs: https://github.com/Stvad/link-summoner#whitelisting-more-websitesadding-custom-preview-types

Reply:
Hm, this doesn't seem to work in NextJS because of initShadowRoot(): https://github.com/Stvad/link-summoner/issues/3

Reply:
(Note to self: Vlad fixed this now!)